src/preload-hover.js

```diff
@@ -12,20 +12,23 @@ export default class PreloadHover {
   }
 
   start(domScope = this.configuration.defaultDomScope) {
-    if (!domScope) { throw new Error('domScope must be provided.'); }
+    if (!domScope) { throw new Error('domScope must be provided.'); } 
+    domScope = [...new Set(domScope)];
```

Comment:
For a page:

```html
<html>
  <head></head>
  <body>
    <div id="container"></div>
  </body>
</html>
```

With the DOM scopes of `[ document.body, document.getElementById('container') ]` would result in duplicates in the links collection. I assume it would apply two event listeners, when only one is desired (although, I haven't tested it to be sure).